### PR TITLE
OADP: Add periodic virtualization tests.

### PR DESCRIPTION
**Pull request by mrnold**
Add 4.16-only periodic OADP virtualization tests to oadp-1.4 and master.

### CODE REVIEW

File: ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master__4.16.yaml
Comment (mateusoliveira43) [inline ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master__4.16.yaml:1]: do not forget to add badge to README in OADP repo after merge

I think is a easy way to keep track of successful/failure rate of periodic tests

**Comment by mrnold** [comment]
/pj-rehearse periodic-ci-openshift-oadp-operator-master-4.16-e2e-test-kubevirt-aws-periodic

**Comment by mrnold** [comment]
/pj-rehearse periodic-ci-openshift-oadp-operator-master-4.16-e2e-test-kubevirt-aws-periodic

**Review by mateusoliveira43 — APPROVED** [review]

**Review by weshayutin — APPROVED** [review]
/LGTM

**Comment by kaovilai** [comment]
resolve conflicts and reherase ack whenever you are ready.

**Comment by mrnold** [comment]
/pj-rehearse periodic-ci-openshift-oadp-operator-master-4.16-e2e-test-kubevirt-aws-periodic periodic-ci-openshift-oadp-operator-oadp-1.4-4.16-e2e-test-kubevirt-aws-periodic

**Review by weshayutin — APPROVED** [review]
/LGTM

**Pull request merged**